test/lib/ex_aws/s3/upload_test.exs

```diff
@@ -22,6 +22,14 @@ defmodule ExAws.S3.UploadTest do
       assert_received :chunk_uploaded
       assert_received :completed_upload
     end
+
+    test "uploading a file with a name separated by spaces" do
+      %{status_code: status_code} =
+        ExAws.S3.put_object("sergey-test", "some file name.txt", "data")
+        |> ExAws.request!
```

Comment:
I changed `sergey-test` bucket to `my-bucket` like in test: `test "uploading a file with a stream"`